www-src/app/change/change_manager.coffee

```diff
@@ -0,0 +1,43 @@
+FileManager = require "./file_manager"
+log = require('../lib/persistent_log')
+    prefix: "Change Manager"
+    date: true
+
+###*
+ * ChangeManager allows to launch the good manager with specific state
+ *
+ * @class ChangeManager
+###
+module.exports = class ChangeManager
+
+    ###*
+     * Create a FilterManager.
```

Comment:
The cookies are in a box labeled "Basmati Rice". It's a metaphor to say that the comment and the code don't match. The comment speaks about `FilterManager` but, in the code, it's a `ChangeManager`.